src/README.md

```diff
@@ -127,6 +127,14 @@ file inside your source code repository.
     and Pipfile.lock.{% if spec.version in ["2.7"] %} (Implies `UPGRADE_PIP_TO_LATEST` to satisfy dependencies of
     Pipenv.){% endif %}
 
+{% if spec.version.startswith("3.") %}
+* **ENABLE_MICROPIPENV**
+
+    Set this variable to use [micropipenv](https://github.com/thoth-station/micropipenv),
+    a lightweight wrapper for pip to support requirements.txt, Pipenv and Poetry lock
+    files or converting them to pip-tools compatible output. Designed for containerized Python applications.
+    Available only for Python 3 images.
```

Comment:
Do we need to expliicitly mention this if the section is only generated for python 3 images?

Reply:
Some people might read e.g. python36 README and then expect it to work on python27, I would say it's worth keeping for clarity.

Reply:
I agree with Tomáš. I'd keep it there.